src/taipy/core/_repository/_sql_repository.py

```diff
@@ -354,7 +364,17 @@ def __to_entity(self, entry: _TaipyVersion) -> Entity:
     def __model_to_entity(self, file_content):
         data = json.loads(file_content, cls=_CustomDecoder)
         model = self.model.from_dict(data)  # type: ignore
-        return self._from_model(model)
+        entity = self._from_model(model)
```

Comment:
I believe the _TaipyVersionTable does not require the migration since there was no version entity on 2.0 yet.